qt.css

```diff
@@ -130,8 +130,8 @@ QLineEdit#mSearchBar {
 }
 
 QLineEdit#mSearchBar QLabel {
-    min-width: 80px;
-    max-width: 80px;
+    min-width: 90px;
+    max-width: 90px;
```

Comment:
Why do you set min width = max width? Depending on the language the length of the string can largely differ.

I suspect that the text is left aligned thus is would be the best to have the box as small as possible but as large as needed with the maximum being just a few pixels smaller than the whole box.

Reply:
min should be 80 and max 100 (or even large)?

Reply:
Unfortunately I cannot test it, but would expect better results with something like

```
    text-align: center;
    margin-left: 16px;
    margin-right: 16px;
```

Reply:
Thanks for advice, I will test it and look forward to better results